src/middlewares/error.middleware.ts

```diff
@@ -45,7 +45,7 @@ const globalErrorHandler = (
 ) => {
   logger.errorRequest(
     req,
-    `An error has occurred. Re-routing to the error screen - ${err.message}`
+    `An error has occurred. Re-routing to the error screen - ${err.stack}`
```

Comment:
noticed that when i had an error it didn't display the stack trace so hard to know what lne of code caused the error. Changed to stack as per ROE, should help with live issues.